### DISCUSSION TITLE
手机上绳网究竟如何发图片

### DISCUSSION THREAD

**Discussion by Tang-Li-1** [share121/inter-knot#1205] [其他]
手机上怎么复制图片啊

**Comment by dont-continue** [comment]
![Screenshot_20240811_061707](https://github.com/user-attachments/assets/0971baec-fca8-438d-8b00-bb222724ccb7)
点击这个，然后找到图片直接点击它(图片一般会在文件中这个需要你自己去找典型的就是picture文件夹或者是camera文件夹需要注意的是有些类似于截图什么的可能会没有文件夹，直接出现在系统列表这个需要自己去找)

**Reply by dont-continue** [reply]
我也不是很专业对手机文件方面不太了解，大致是这样吧我自己就是这么找到的

**Reply by Tang-Li-1** [reply]
解决了，谢谢

**Comment by Tang-Li-1** [comment]
感谢 已解决

**Comment by zifeiyu1111** [comment]
绳网手机端怎么整？

**Comment by lllliufeng** [comment]
现在帖子显示楼主了？（wow

**Reply by lllliufeng** [reply]
还有自己
